package.json

```diff
@@ -4,7 +4,7 @@
   "displayName": "R Code Analyzer (flowR)",
   "description": "Slicing and analyzing R code courtesy of flowR",
   "homepage": "https://github.com/flowr-analysis/vscode-flowr/blob/main/README.md",
-  "icon": "media/flowR.png",
+  "icon": "resources/flowR.png",
```

Comment:
i do not remember anymore, but does this break a documentation link? + i weakly remember that there was something against resources but i do not remember what it was

Reply:
the flowr logo is still also present in `media`, it's just that we need to bundle only what is important for the actual extension, and the png is used as the logo of the dependencies view :)